src/services/composite-keys-manager.js

```diff
@@ -1,11 +1,23 @@
 import _ from 'lodash';
 import Operators from '../utils/operators';
 
+const REGEX_UUID = /[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}/g;
+
 function CompositeKeysManager(model, schema, record) {
   const GLUE = '-';
 
   this.getPrimaryKeyValues = function getPrimaryKeyValues(recordId) {
-    const primaryKeyValues = recordId.split(GLUE);
+    let primaryKeyValues = recordId.split(GLUE);
+    const modelPrimaryKeys = model && model.primaryKeys ? _.keys(model.primaryKeys) : [];
+    const primaryKeyValuesMatchingUUID = recordId.match(REGEX_UUID) || [];
```

Comment:
There might be an issue if only one key is a UUID in the composite key.

Reply:
Nice catch ... I added some tests case for this one, which obviously fail.
It mean that I **need** to check for types in this method.

Reply:
After a small talk with @arnaudbesnier, the solution will just be to change the `GLUE` character to something with less probability of occurring in a key composing the composite key. Will close this PR and reopen one with just the glue char changed